### ISSUE TITLE
Add Chaos by project discovery

### ISSUE THREAD

**Issue by rudiandriano** [Screetsec/Sudomy#82]
something the best now is http://chaos.projectdiscovery.io/ , maybe you want to try

**Comment by Screetsec** [comment]
We don't know what the resource they are used to collect subdomains. 
For example, if chaos using shodan, dnsdb, censys, hacktertarget and etc as resources,so that will collect subdomains in 2x repetitions which the results are same. Therefore I am still considering about i

**Comment by rudiandriano** [comment]
Ya, looks like they have some tools to collect and compile the result on the Database, what my experience the result always good than some tools i using currently

**Comment by Screetsec** [comment]
Yeah, it's natural to want to compare it. But thanks for the suggestion, I will consider adding it.

**Comment by rudiandriano** [comment]
Thanks a lot Brother!
